web/modules/custom/hoeringsportal_deskpro/src/Service/HearingHelper.php

```diff
@@ -414,6 +382,142 @@ public function synchronizeHearingTickets(Node $hearing) {
     return ['hearing' => $hearing->id(), 'data' => $data];
   }
 
+  /**
+   * Get data from Deskpro and store in hearing node.
+   */
+  public function synchronizeTicket(array $payload, $delayed = TRUE) {
+    [$hearing, $ticketId] = $this->validateTicketPayload($payload);
```

Comment:
`validateTicketPayload` may throw `RuntimeException` are these handled?

Reply:
Not really, but we don't really care about handling invalid payloads in a nice way, i.e. we expect them to be valid.